orders/tasks.py

```diff
@@ -0,0 +1,16 @@
+from celery import task
+from django.core.mail import send_mail 
+from .models import Order
+
+@task
+def order_created(order_id):

```

Comment:
Task name order_created is not clear. 
![image](https://github.com/kadhir-testpress/online-shop-main/assets/89839073/7b50d096-fe58-4ad0-ac08-30d9a9ba0b34)